qiskit_ibm/ibm_provider.py

```diff
@@ -60,20 +59,14 @@ class IBMProvider(Provider):
         from qiskit_ibm import IBMProvider
         IBMProvider.save_account(token=<INSERT_IBM_QUANTUM_TOKEN>)
 
-    The open access provider (`ibm-q/open/main`) is the default provider, but you can overwrite
+    The open access project (`ibm-q/open/main`) is the default, but you can overwrite
```

Comment:
It's not really the default anymore right? Since the default is a premium hgp unless the user only has open.